README.md

```diff
@@ -37,7 +64,16 @@ A [tutorial](https://github.com/bids-standard/bids-matlab/blob/master/examples/t
 
 ## Requirements
 
-BIDS-MATLAB works with MATLAB R2014a or newer, or Octave 4.2.2 or newer. (It may also work with older versions, but those are not actively supported.)
+BIDS-MATLAB works with: 
+
+- Octave 4.2.2 or newer
+- MATLAB R2014a or newer  
+
+In general we will try to use the latest version of Octave as a baseline for compatibility.
```

Comment:
"We aim for compatibility with the latest stable release of Octave at any time. Compatibility can sometimes also be achieved with older versions of Octave but this is not guaranteed."

Reply:
```suggestion
We aim for compatibility with the latest stable release of Octave at any time. Compatibility can sometimes also be achieved with older versions of Octave but this is not guaranteed.
```